pinot-core/src/main/java/org/apache/pinot/core/operator/transform/TransformOperator.java

```diff
@@ -39,28 +37,23 @@
  * Class for evaluating transform expressions.
  */
 public class TransformOperator extends BaseOperator<TransformBlock> {
-
   private static final String OPERATOR_NAME = "TransformOperator";
 
   private final ProjectionOperator _projectionOperator;
   private final Map<String, DataSource> _dataSourceMap;
   private final Map<TransformExpressionTree, TransformFunction> _transformFunctionMap = new HashMap<>();
-  private final List<TransformExpressionTree> _expressions;
 
   /**
    * Constructor for the class
    *
    * @param projectionOperator Projection operator
    * @param expressions Set of expressions to evaluate
    */
-  public TransformOperator(@Nonnull ProjectionOperator projectionOperator,
-      @Nonnull List<TransformExpressionTree> expressions) {
+  public TransformOperator(ProjectionOperator projectionOperator, Collection<TransformExpressionTree> expressions) {
```

Comment:
I see this PR is removing `@Nonnull` annotations. If this is not related to this change, it would be good to take it out of this PR, or list down all such changes (along with the intention), in the PR description. It will make it easier to review the PR.